src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  datasource:
+    driver-class-name: org.h2.Driver
+    username: sa
+    password:
+    url: jdbc:h2:file:~/to_do
+  jpa:
+    hibernate:
+      ddl-auto: update
+  h2:
+    console:
+      enabled: true
+      path: /h2
+
+logging:
+  config: classpath:log-config.xml
```

Comment:
There is no file called `log-config.xml` on `src/main/resources` folder, please add the configuration file or remove the entry